package proto_ff;

import "yd_fieldoptions.proto";


message chatchat
{
	optional int64 chatId = 1[(yd_fieldoptions.field_cname) = "传闻ID"];
	optional int32 chatType = 2[(yd_fieldoptions.field_cname) = "传闻类型"];
	optional int32 showTime = 3[(yd_fieldoptions.field_cname) = "活动提示显示时间"];
	optional string chatContent = 4[(yd_fieldoptions.field_cname) = "传闻内容", (yd_fieldoptions.field_bufsize) = 157];
	optional int32 channel = 5[(yd_fieldoptions.field_cname) = "显示频道"];
	optional int32 CrossService = 6[(yd_fieldoptions.field_cname) = "是否跨服广播"];
}


message Sheet_chatchat
{
	repeated chatchat chatchat_List = 1[(yd_fieldoptions.field_arysize)=296];
}

message chatspecialchat
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "事件类型"];
	optional string parama = 2[(yd_fieldoptions.field_cname) = "事件参数a", (yd_fieldoptions.field_bufsize) = 32];
	optional string paramb = 3[(yd_fieldoptions.field_cname) = "事件参数b", (yd_fieldoptions.field_bufsize) = 71];
	optional int64 chat_Id = 4[(yd_fieldoptions.field_cname) = "触发关联的跑马灯内容id"];
}


message Sheet_chatspecialchat
{
	repeated chatspecialchat chatspecialchat_List = 1[(yd_fieldoptions.field_arysize)=111];
}
